lib/qless/job.rb

```diff
@@ -63,6 +63,9 @@ def perform
         return fail("#{queue_name}-NameError", "Cannot find #{klass_name}")
       end
 
+      # log a real process executing job -- before we start processing
+      log("started by pid:#{Process.pid}")
```

Comment:
I can ague it both ways. 
* It should be a middleware -- it creates a bit of noise, which is not universally needed.
* It should be hardcoded -- The location is chosen  after preliminary checks (like klass) are done, but before any middleware (custom code) runs. Potentially, we might have trouble code in middleware. Knowing process would help to troubleshoot it. In addition, 'started' constitutes a stage in job's lifecycle very similar to popped, failed, or completed.

I am a bit more inclined toward second option. May be, we should consider configuration-driven verbosity of logging (e.g., logging stages of middleware, etc in some debug mode). I am completely open to re-do it as soon as we get some experience with the feature.